Sources/Bugsnag/PayloadTransformer.swift

```diff
@@ -3,17 +3,24 @@ import Stacked
 import HTTP
 
 public protocol PayloadTransformerType {
-    func payloadFor(message: String, metadata: Node?, request: Request?) throws -> JSON
+    func payloadFor(
+        message: String,
+        metadata: Node?,
+        request: Request?,
+        severity: Severity
+    ) throws -> JSON
 }
 
 internal struct PayloadTransformer: PayloadTransformerType {
     let drop: Droplet
     let config: ConfigurationType
-
+    
+    
     internal func payloadFor(
         message: String,
         metadata: Node?,
-        request: Request?
+        request: Request?,
+        severity: Severity = Severity.error
```

Comment:
I'm not convinced that the payload transformer should include a default value. I think it makes more sense for the reporter to have/know this. What do you think?